doc/index.rst

```diff
@@ -71,6 +71,11 @@ Features
 
 ..
 
+- *Just in time compilation*. Compile your **entire hybrid workflow**, including loops. See
```

Comment:
@trbromley should we make the same edit here as in the readme, or would that be too much text?
```suggestion
- *Just in time compilation*. Compile your **entire hybrid workflow**, with support for
  adaptive circuits, real-time measurement feedback, unbounded loops, and more. See
```